doc/examples/features_detection/plot_shape_index.py

```diff
@@ -150,7 +150,7 @@ def create_test_image(
     **scatter_settings
 )
 
-ax2.legend()
```

Comment:
Loc can also be a word, right? 3 is hard to interpret.

Reply:
yep!